environment-pipelines/buildspec.yml

```diff
@@ -35,7 +35,11 @@ phases:
 
   build:
     commands:
-      - echo "Build Phase"
+      - echo "Terraform Plan Phase"
```

Comment:
Should these commands fall under the `pre_build` phase?

Reply:
Yes probably once the rest of the copilot and platform-helper commands are added. This stuff is most likely going to be moved to a separate buildspec file anyway to work with a plan -> manual approve -> apply process